Parallel make fixes

Patch by Kevin McCarthy <signals@gentoo.org>

--- Makefile.noimake
+++ Makefile.noimake
@@ -21,12 +21,12 @@
 
 all: explorer.o image.o xrmap.o
 	gcc -o xrmap explorer.o image.o xrmap.o $(LDOPTIONS)
-	cd editkit ; make -f Makefile.kit DESTDIR=$(DESTDIR)
-	cd earthview ; make DESTDIR=$(DESTDIR)	
+	$(MAKE) -C editkit -f Makefile.kit DESTDIR=$(DESTDIR)
+	$(MAKE) -C earthview DESTDIR=$(DESTDIR)	
 	@cat EXTRADATA
 	
 numdefs.h:
-	cd tools ; make
+	$(MAKE) -C tools
 
 explorer.o: xrmap.h version.h bitmaps.h numdefs.h menudefs.h xrmap.c
 	gcc $(CCOPTIONS) -c explorer.c
--- tools/Makefile
+++ tools/Makefile
@@ -3,16 +3,16 @@
 	./preproc > ../numdefs.h
 
 locutils:
-	cd locutils; make
+	$(MAKE) -C locutils
 
 cbd2else:
-	cd cbd2else; make
+	$(MAKE) -C cbd2else
 
 jpd2else:
-	cd jpd2else; make
+	$(MAKE) -C jpd2else
 
 rez2else:
-	cd rez2else; make
+	$(MAKE) -C rez2else
 
 world: locutils cbd2else jpd2else rez2else
 
